static/_redirects

```diff
@@ -496,8 +495,12 @@
 /docs/admin/authorization/webhook/     /docs/reference/access-authn-authz/webhook/ 301
 /docs/admin/authorization/     /docs/reference/access-authn-authz/authorization/ 301
 /docs/admin/high-availability/building/     /docs/setup/production-environment/tools/kubeadm/high-availability/   301
+
 /code-of-conduct/     /community/code-of-conduct/   301
 /values/     /community/values/   302
+/dockershim /blog/2020/12/02/dockershim-faq/ 302
+/dockershim/ /blog/2020/12/02/dockershim-faq/ 302
```

Comment:
Replacing an alias set in `content/en/blog/_posts/2020-12-02-dockershim-faq.md`

Reply:
I used a 302 redirect (not 301) in case we want to change where we direct people to.